deploy/helm/templates/config.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: security-profiles-operator.x-k8s.io/v1alpha1
+kind: SecurityProfilesOperatorDaemon
+metadata:
+  labels:
+    app: security-profiles-operator
+  name: spod
+  namespace: security-profiles-operator
+spec:
+  enableSelinux: {{ .Values.enableSelinux }}
+  enableLogEnricher: {{ .Values.enableLogEnricher }}
+  enableAppArmor: {{ .Values.enableAppArmor }}
```

Comment:
I would expose also other flags in here such as: `enableBpfRecoder`, `verbosity`, `enableProfiling`. The same all switched off by default.

Reply:
`+1` considering the title of the PR I think that would make sense.